Kinvey/Kinvey/HttpRequest.swift

```diff
@@ -307,9 +313,7 @@ internal class HttpRequest: TaskProgressRequest, Request {
         self.endpoint = Endpoint.url(url: request.url!)
         self.client = client
         
-        if let authorization = request.value(forHTTPHeaderField: HttpHeader.authorization(credential: nil).name) {
-            self.credential = HttpHeaderCredential(authorization)
-        } else {
+        if request.value(forHTTPHeaderField: HttpHeaderKey.authorization.rawValue) == nil {
```

Comment:
@heyzooi If the value is not `nil`, will `self.credential` be set correctly? The old and new code don't seem to match up, hence the question.